profiles/common/modules/custom/nexteuropa_token/src/Entity/ViewModeTokenHandler.php

```diff
@@ -79,6 +79,13 @@ public function hookTokens($type, $tokens, array $data = array(), array $options
                 $this->watchdogTokenNotFound($data, $original);
               }
               break;
+
+            case 'bean':
+              $bean = bean_load($entity_id);
+              if (user_access('access user profiles')) {
```

Comment:
Certainly not 'access user profiles'. You seem to have just copy-pasted that from the code a bit higher.

```
case 'user':
  if ($account = user_load($entity_id)) {
    if (user_access('access user profiles')) {
      $render = user_view($account, $view_mode);
       ...
   }
```